source/stylesheets/refills/unstyled/_button-group.scss

```diff
@@ -0,0 +1,30 @@
+.button-group {  
+  $large-screen: em(860) !default;
+
+  input {
+    display: none;
+  }
+
+  label {
+    font-weight: normal;
+
+    @include media($large-screen) {
+      padding: 0 0.5em;
+      float: left;
+    }
+
+    .button-group-item {
+      cursor: pointer;
+      display: inline-block;
+      width: 100%;
+
+      @include media($large-screen) {
+        width: auto;
+      }
+    }
+
+    input:checked + .button-group-item {
+      color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.